### PR TITLE
fix: use optional chain operator

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

**Comment by jobo322** [comment]
@targos is it ok?

File: src/util/internalPeaks/getInternalPeaks.ts [hunk 1]
```diff
@@ -134,3 +132,15 @@ function getNormalizedValue(
   }
   return value;
 }
+
+function getParameterByKey<
+  Parameters extends Record<Parameter, InitialParameter>,
+>(
```
Comment (targos) [inline src/util/internalPeaks/getInternalPeaks.ts:138]: Not sure why you created a function for it? It seems easier to understand if the optional chaining is inline.

Comment (jobo322) [reply src/util/internalPeaks/getInternalPeaks.ts:138]: It is true, I did like it

**Comment by targos** [comment]
Node 12 is not supported anymore so you can just remove it from the github worflow and add node 18 instead

**Pull request merged**